smartrouter/image.yaml

```diff
@@ -0,0 +1,27 @@
+name: "jboss-bpmsuite-7/bpmsuite70-smartrouter"
+description: "Red Hat JBoss BPM Suite 7.0 Smart Router container image"
+version: "7.0.0"
+from: "jboss/openjdk18-rhel7:1.0"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "bpmsuite"
```

Comment:
In my opinion `bpmsuite-smartrouter` follows the product-subproduct naming convention. I've used it